evalml/tests/data_checks_tests/test_outliers_data_check.py

```diff
@@ -86,3 +86,39 @@ def test_outliers_data_check_string_cols():
                                       details={"columns": ["d"]}).to_dict()],
         "errors": []
     }
+
+
+def test_outlier_score():
+    a = np.arange(10) * 0.01
+    data = np.tile(a, (100, 10))
+
+    X = pd.DataFrame(data=data)
+    X.iloc[0, 3] = 1000
+    X.iloc[3, 25] = 1000
+    X.iloc[5, 55] = 10000
+    X.iloc[10, 72] = -1000
+
+    for col in X.columns:
+        results = OutliersDataCheck._outlier_score(X[col], convert_column=False)
+        if col in [3, 25, 55, 72]:
+            assert results['score'] != 1.0
+            assert len(results['values']['high_values']) != 0 or len(results['values']['low_values']) != 0
+        else:
+            assert results['score'] == 1.0
+            assert len(results['values']['high_values']) == 0 and len(results['values']['low_values']) == 0
+
+
+def test_outlier_score_convert_column_to_int():
+    has_outlier = pd.Series(np.append(np.arange(10), 1000)).astype(object)
+    results = OutliersDataCheck._outlier_score(has_outlier, convert_column=True)
+    assert results['score'] != 1.0
+    len(results['values']['high_values']) != 0 or len(results['values']['low_values']) != 0
+    no_outlier = pd.Series(np.arange(10)).astype(object)
+    results = OutliersDataCheck._outlier_score(no_outlier, convert_column=True)
+    assert results['score'] == 1.0
+    len(results['values']['high_values']) == 0 and len(results['values']['low_values']) == 0
+
+
+def test_outlier_score_all_nan():
```

Comment:
Yea, I think this might be a bit repetitious as we already have a data check for this.  Do we check for all nans in other DataChecks?